android/build.gradle

```diff
@@ -21,7 +21,7 @@ def safeExtGet(prop, fallback) {
 android {
     compileSdkVersion safeExtGet('FileUtils_compileSdkVersion', 29)
     defaultConfig {
-        minSdkVersion safeExtGet('FileUtils_minSdkVersion', 16)
+        minSdkVersion safeExtGet('FileUtils_minSdkVersion', 24)
```

Comment:
What is the reason for SDK 24 here? The app currently is on 21, I'm not against upgrading but we would need to run some analytics first to see how many of our users are below 24.

Reply:
Oh yes, it was only an issue with version 16 and some methods we needed. I think 21 should work fine. Lowering that down a bit.